input/vulnerabilities/Webkit-Use-After-Free.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name" : "WebKit Use-After-Free",
+  "CVE" : [["CVE-2010-1119"]],
+  "Coordinated_disclosure" : "unknown",
+  "Categories" : ["system", "app"],
+  "Details" : [["A vulnerability in the WebKit browser engine allows a malicious webpage to perform remote code execution"]],
+  "Discovered_by" : [["MJ Keith", "exploit-db"]],
+  "Discovered_on" : [["2011-03-11", "exploit-db"]],
+  "Submission" : [{"by": "dcc52", "on": "2019-07-03"}],
+  "Reported_on" : [],
+  "Fixed_on" : [],
+  "Fix_released_on" : [],
```

Comment:
This is presumably the release date of 2.1.2?

Reply:
Done (in fact seems to be 2.2 as I don't think 2.1.2 existed)